lib/Spreadsheet/XLSX/SharedStrings.rakumod

```diff
@@ -21,27 +21,27 @@ class Spreadsheet::XLSX::SharedStrings does Positional {
             die X::Spreadsheet::XLSX::Format.new: message =>
                     'Shared strings file did not start with tag sst';
         }
-        self.new(:$root, :backing($sst))
+        self.from-xml-element($sst, :profile{ :$root })
     }
 
     #| Create a new, empty, string table.
     method empty(Spreadsheet::XLSX::Root :$root!--> Spreadsheet::XLSX::SharedStrings) {
         self.new(:$root)
     }
 
-    #| Get the shared string entry at the given position. Creates a fresh
-    #| object each time, which an given sheet can cache as a particular
-    #| cell position.
-    method AT-POS(Int $idx) {
-        with $!backing {
-            with $!backing.childNodes[$idx] -> LibXML::Element $si {
-                if $si.nodeName ne 'si' {
-                    die X::Spreadsheet::XLSX::Format.new: message =>
-                            'Shared strings entry was not an si node';
-                }
-                return shared-cell-from-xml($si.first);
-            }
-        }
-        fail "Could not resolve shared string entry $idx";
-    }
+#    #| Get the shared string entry at the given position. Creates a fresh
+#    #| object each time, which an given sheet can cache as a particular
+#    #| cell position.
+#    method AT-POS(Int $idx) {
+#        with $!backing {
+#            with $!backing.childNodes[$idx] -> LibXML::Element $si {
+#                if $si.nodeName ne 'si' {
+#                    die X::Spreadsheet::XLSX::Format.new: message =>
+#                            'Shared strings entry was not an si node';
+#                }
+#                return shared-cell-from-xml($si.first);
+#            }
+#        }
+#        fail "Could not resolve shared string entry $idx";
+#    }
```

Comment:
Would prefer to not have left-over commented out code.

Reply:
Argh... Sorry, one of the situations when you think "ok, will do it later"...

Reply:
Perhaps remove it when fixing up the other PR (which has a conflict now).